opentimelineview/timeline_widget.py

```diff
@@ -537,6 +544,186 @@ def wheelEvent(self, event):
+        curItemXpos = curSelectedItem.pos().x()
+
+        if curSelectedItem.parentItem():
+            curTrackYpos = curSelectedItem.parentItem().pos().y()
+
+            newXpos = curItemXpos
+            newYpos = curTrackYpos - TRACK_HEIGHT
+
+            newSelectedItem = self.scene().itemAt(
+                                                  QtCore.QPointF(
+                                                                 newXpos,
+                                                                 newYpos
+                                                                 ),
+                                                  QtGui.QTransform()
+                                                  )
+
+            if not newSelectedItem or isinstance(newSelectedItem, Track):
+                newYpos = newYpos - TRANSITION_HEIGHT
+        else:
+            newXpos = curItemXpos
+            newYpos = curSelectedItem.y()
+
+        newPosition = QtCore.QPointF(newXpos, newYpos)
+
+        return self.scene().itemAt(newPosition, QtGui.QTransform())
+
+    def _get_down_item(self, curSelectedItem):
+        curItemXpos = curSelectedItem.pos().x()
+
+        if curSelectedItem.parentItem():
+            curTrackYpos = curSelectedItem.parentItem().pos().y()
+            newXpos = curItemXpos
+            newYpos = curTrackYpos + TRACK_HEIGHT
+
+            newSelectedItem = self.scene().itemAt(
+                                                  QtCore.QPointF(
+                                                                 newXpos,
+                                                                 newYpos
+                                                                 ),
+                                                  QtGui.QTransform()
+                                                  )
+
+            if not newSelectedItem or isinstance(newSelectedItem, Track):
+                newYpos = newYpos + TRANSITION_HEIGHT
+
+            if newYpos < TRACK_HEIGHT:
+                newYpos = TRACK_HEIGHT
+        else:
+            newXpos = curItemXpos
+            newYpos = MARKER_SIZE + TIME_SLIDER_HEIGHT + 1
+            newYpos = TIME_SLIDER_HEIGHT
+        newPosition = QtCore.QPointF(newXpos, newYpos)
+
+        return self.scene().itemAt(newPosition, QtGui.QTransform())
+
+    def _select_new_item(self, newSelectedItem):
+        # Check for text item
+        # Text item shouldn't be selected,
+        # maybe a bug in the population of timeline.
+        if isinstance(newSelectedItem, QtWidgets.QGraphicsSimpleTextItem):
+            newSelectedItem = newSelectedItem.parentItem()
+
+        # Validate new item for edge cases
+        # If valid, set selected
+        if (
+            not isinstance(newSelectedItem, Track)
+            and newSelectedItem
+        ):
+            if self.scene().selectedItems():
+                for selectedItem in self.scene().selectedItems():
+                    selectedItem.setSelected(False)
```

Comment:
Could you either put this into a `_deselect_all()` method or just comment that you're deselecting everything?

Reply:
np! _deselect_all() sounds good to me.